### PR TITLE
[FIX] Notification preferences of discussions 

### PR DESCRIPTION
**Pull request by devyaniChoubey**
After fix discussion name is showing
solve issue #1842 
Screenshot_20200307-224449_ DEVELOP  RocketChatRN

### CODE REVIEW

File: app/views/NotificationPreferencesView/index.js [hunk 1]
```diff
@@ -252,7 +252,7 @@ class NotificationPreferencesView extends React.Component {
 						theme={theme}
 					/>
 					<Separator theme={theme} />
-					<Info info={I18n.t('Receive_notifications_from', { name: room.name })} theme={theme} />
+					<Info info={I18n.t('Receive_notifications_from', { name: room.prid ? room.fname : room.name })} theme={theme} />
```
Comment (djorkaeffalexandre) [inline app/views/NotificationPreferencesView/index.js]: Instead it, use `RocketChat.getRoomTitle`.

Comment (devyaniChoubey) [reply app/views/NotificationPreferencesView/index.js]: @djorkaeffalexandre done

Comment (devyaniChoubey) [reply app/views/NotificationPreferencesView/index.js]: @djorkaeffalexandre Please review

**Comment by Prateek93a** [comment]
@devyaniChoubey  The same random name shows when we try to leave the discussion, can you please add a commit to solve that also, Thanks

**Comment by devyaniChoubey** [comment]
> @devyaniChoubey The same random name shows when we try to leave the discussion, can you please add a commit to solve that also, Thanks
> 

@Prateek93a ok

**Pull request closed without merging**